hack/run-e2e-kind.sh

```diff
@@ -402,6 +446,7 @@ trap cleanup EXIT
 update_test_host
 check-prerequisites 
 kind-up-cluster
+extend-resources
```

Comment:
Why not combine the extend-resources functions and setup-mcad-env function ?

Reply:
See comment above. I would consider adding it to the `kind-up-cluster` but I don't think it should be in the `setup-mcad-env` function. My view on this is that the extended resources is just cluster configuration that doesn't involve MCAD. Creating the cluster adds GPUs. That happens even before MCAD comes into the picture, so it shouldn't be considered a "MCAD setup", since it is not.